_data/government.yml

```
@@ -140,6 +140,16 @@ websites:
     regions:
       - au
 
+  - name: Pennsylvania Dept of Revenue
+    url: https://mypath.pa.gov
+    img: PA-revenue.png
```

```suggestion
    img: mypath.png
```
The image names are normally all lowercase. Perhapt mypath.png works better?

---

I didn't know they were all lowercase, I see the Jekyll regex failed. mypath should work fine as well

---

Do I need to re upload the image or can you fix it?

---

I'm on my phone right now so I'm afraid I can't change that at the moment.
If you'd like to remove rename the image that'd be great. If not, I'll do it in a couple of hours 🙂